arlo-client/src/components/AuditFlow/index.tsx

```diff
@@ -37,20 +37,19 @@ const AuditFlow: React.FC<IProps> = ({
   const [audit, setAudit] = useState<IAudit | undefined>(undefined) // here entereth the error monster
```

Comment:
This comment can be removed now that particular error monster hath been slain.

Reply:
Done.